### PR TITLE
Add TATEditor Latest

### PR DESCRIPTION
**Pull request by mitaroThanken**
TATEditor is a text editor that can be vertical writing.

### CODE REVIEW

File: Casks/tateditor.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'tateditor' do
+  version :latest
+  sha256 :no_check
+
+  url 'http://www.cc4966.net/software/tateditor-osx.dmg?attredirects=0&d=1'
```
Comment (vitorgalvao) [inline Casks/tateditor.rb]: The `?attredirects=0&d=1` part seems to not be needed. Please remove it.

**Comment by mitaroThanken** [comment]
I removed the ```?attredirects=0&d=1``` part from URL.

**Comment by vitorgalvao** [comment]
Thank you. Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

File: Casks/tateditor.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'tateditor' do
+  version :latest
+  sha256 :no_check
+
+  url 'http://www.cc4966.net/software/tateditor-osx.dmg'
+  name 'TATEditor'
+  homepage 'http://www.cc4966.net/'
+  license :unknown
```
Comment (vitorgalvao) [inline Casks/tateditor.rb]: Could you add a license, please? Being in Japanese, these would be something hard to figure out down the line.

**Comment by mitaroThanken** [comment]
Changed license to ```gratis``` and squashed commits.

**Comment by vitorgalvao** [comment]
Thank you. Merged.

**Pull request merged**